openapi-cli/src/main/java/org/ballerinalang/openapi/CodeGenerator.java

```diff
@@ -184,7 +183,7 @@ public void generateService(String executionPath, String definitionPath,
 
         if (serviceName != null) {
             api.getInfo().setTitle(serviceName);
-        } else if (api.getInfo() == null || StringUtils.isEmpty(api.getInfo().getTitle())) {
+        } else if (api.getInfo() == null || api.getInfo().getTitle().isBlank()) {
             api.getInfo().setTitle(GeneratorConstants.UNTITLED_SERVICE);
```

Comment:
`setTitle` can produce NPE when `api.getInfo() == null`?

Reply:
yeah , will fix it.